scripts/analysis_0/nbconverted/5_create_figs_manuscript.py

```diff
@@ -0,0 +1,481 @@
+
```

Comment:
aren't these same figures created in other notebooks?

Reply:
These figures are the same similarity curves but now plotting the average svcca score per experiment and showing the error bars.